scripts/get_latest_build.py

```diff
@@ -22,8 +22,8 @@ def detect_latest_date(source, user, passwd):
 
     soup = retrieve_url(source, user, passwd)
     dates = [d.text[:-1] for d in soup.find_all("a", href=re.compile("..-..-.."))]
-    last_date = dates[0]
-    print "Latest date: {}".format(last_date)
+    last_date = dates
+    print "Latest date: {}".format(last_date[0])
     return last_date
```

Comment:
Yes, you're right, I thought "I will change it as soon as I've tested it"... but I forgot :dancer: